src/newViews/MyProfile/index.tsx

```diff
@@ -1,11 +1,287 @@
+                    />
+                    <div className={styles.userInfo}>
+                        <Container
+                            className={styles.personalInfo}
+                            heading={_ts('myProfile', 'personalInfo')}
+                            sub
+                        >
+                            <TextInput
+                                name="firstName"
+                                disabled={disabled}
+                                onChange={onValueChange}
+                                value={value?.firstName}
+                                error={error?.fields?.firstName}
+                                label={_ts('myProfile', 'firstName')}
+                                placeholder={_ts('myProfile', 'firstName')}
+                                autoFocus
+                            />
+                            <TextInput
+                                name="lastName"
+                                disabled={disabled}
+                                onChange={onValueChange}
+                                value={value?.lastName}
+                                error={error?.fields?.lastName}
+                                label={_ts('myProfile', 'lastName')}
+                                placeholder={_ts('myProfile', 'lastName')}
+                            />
+                            <TextInput
+                                name="organization"
+                                disabled={disabled}
+                                onChange={onValueChange}
+                                value={value?.organization}
+                                error={error?.fields?.organization}
+                                label={_ts('myProfile', 'organization')}
+                                placeholder={_ts('myProfile', 'organization')}
+                            />
+                        </Container>
+                        <Container
+                            className={styles.preferences}
+                            sub
+                            heading={_ts('myProfile', 'preferences')}
+                        >
+                            <SelectInput
+                                name="language"
+                                disabled={disabled}
+                                onChange={onValueChange}
+                                value={value?.language}
+                                error={error?.fields?.language}
+                                label={_ts('myProfile', 'platformLanguage')}
+                                placeholder={_ts('myProfile', 'platformLanguage')}
+                                keySelector={langaugeKeySelector}
+                                labelSelector={languageLabelSelector}
+                                options={languageResponse?.results}
+                            />
+                            <List // FIXME:  use ListSelection component when available
```

Comment:
Did we create a task to use ListSelection on MyProfile form? If so, we can merge this @samshara

Reply:
yes I did create a task on zube. If all is good we can merge this @tnagorra .

Reply:
sure!